### PR TITLE
Make fixes

### LINKED ISSUE #8: Make ArrayList helper methods private

### PR DESCRIPTION
**Pull request by tnsyn**
Resolve #8 #7 #6 #5 #4 #2 #1

### CODE REVIEW

File: lists/ArrayList.py [hunk 1]
```diff
@@ -1,119 +1,170 @@
...
+
+
 class ArrayList:
-    # Constructor that takes in a capacity, which is the current maximum size of the ArrayList.
+    """Constructor that takes in a capacity, which is the current maximum size of the ArrayList."""
+
     def __init__(self, capacity=1000):
         self.capacity = capacity
-        self.array_list = [None] * capacity
+        self.array_list = [None for x in range(capacity)]
```
Comment (leeyiheng12) [inline lists/ArrayList.py:9]: Since `x` is unused, can consider `[None for _ in range(capacity)]`

File: lists/ArrayList.py [hunk 2]
```diff
@@ -1,119 +1,170 @@
...
+    Returns True if the ArrayList contains the item and False otherwise.
+    Time complexity: O(n).
+    """
+
     def contains(self, item):
-        for i in range(self.size):
-            if item == self.array_list[i]:
-                return True
-        return False
+        return any(x == item for x in self.array_list)
```
Comment (leeyiheng12) [inline lists/ArrayList.py:45]: NICE

File: lists/ArrayList.py [hunk 3]
```diff
@@ -1,119 +1,170 @@
...
-                item_string = str(self.array_list[i])
-            else:
-                item_string = ", ".join([item_string, str(self.array_list[i])])
+    """
+    Prints out all elements in the ArrayList.
+    Time complexity: O(n).
+    """
+
+    def print_list(self):
+        item_string = ", ".join(str(x) for x in self.array_list if x is not None)
```
Comment (leeyiheng12) [inline lists/ArrayList.py:149]: NICE

**Review by leeyiheng12 — APPROVED** [review]
NICE

File: lists/ArrayList.py (hunk 1 above)
Comment (tnsyn) [reply lists/ArrayList.py]: Changed

**Pull request merged**